catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -713,43 +725,54 @@ private int updateRecords(UpdateAction updateAction)
             queryRequest = queryFactory.updateQueryRequestTags(queryRequest,
                     schemaTransformerManager.getTransformerSchemaForId(updateAction.getTypeName()));
 
-            QueryResponse response = framework.query(queryRequest);
+            Map<String, Serializable> recordProperties = updateAction.getRecordProperties();
+            Iterable<List<Result>> resultList = Iterables.partition(ResultIterable.resultIterable(
+                    framework,
+                    queryRequest), 64);
+            AtomicInteger batch = new AtomicInteger(1);
```

Comment:
✏️  Should we rename `batch` to `batchCount` or `batchNumber` to make it clear that this is _not_ the batch of results?